### PR TITLE
chore: add rate type to pump model result

### PR DESCRIPTION
**Pull request by frodehk**
## Why is this pull request needed?

Missing rate type (stream day or calendar day) for `PumpModelResult`

## What does this pull request change?

- [x] Add rate type for `PumpModelResult` in `get_asset_result`
- [x] Convert to time series
- [x] Update snapshots

## Issues related to this change:
https://equinor-ecalc.atlassian.net/browse/ECALC-240?atlOrigin=eyJpIjoiYTczYzE0YTgxYjMxNDMwMDk2MTFiMmQ3NjcyNGI5ZWMiLCJwIjoiaiJ9

### CODE REVIEW

File: src/libecalc/common/units.py [hunk 1]
```diff
@@ -49,6 +49,7 @@ class Unit(str, Enum):
     KG_SM3 = "kg/Sm3"
     KG_M3 = "kg/m3"
     STANDARD_CUBIC_METER = "Sm3"
+    CUBIC_METER = "m3"
```
Comment (TeeeJay) [inline src/libecalc/common/units.py:52]: Sm3 is for compressors, m3 is for pumps? Also not Am3?

**Comment by TeeeJay** [comment]
Seems ready, is it @frodehk ?

**Comment by TeeeJay** [comment]
@jsolaas

File: src/libecalc/common/units.py (hunk 1 above)
Comment (frodehk) [reply src/libecalc/common/units.py:52]: Rate unit is Am3 both for pump- and compressor models (spec in yaml). However, the rate vector in models for compressor is `rate_sm3_day`, while it is  `inlet_liquid_rate_m3_per_day` for pumps. That is why i added CUBIC_METER.

Comment (frodehk) [reply src/libecalc/common/units.py:52]: Have changed to Sm3/d".

File: src/libecalc/core/graph_result.py [hunk 2]
```diff
@@ -649,6 +650,59 @@ def get_asset_result(self) -> libecalc.dto.result.EcalcModelResult:
...
+                                    if model.outlet_pressure_bar is not None
+                                    else [math.nan] * len(model.timesteps),
+                                    unit=Unit.BARA,
+                                ),
+                                operational_head=TimeSeriesFloat(
+                                    timesteps=model.timesteps,
+                                    values=model.operational_head
+                                    if model.operational_head is not None
+                                    else [math.nan] * len(model.timesteps),
+                                    unit=Unit.POLYTROPIC_HEAD_KILO_JOULE_PER_KG,  # Docs: support for M, KJ/KG and J/KG: how to check?
```
Comment (frodehk) [inline src/libecalc/core/graph_result.py:700]: @TeeeJay @jsolaas : Docs states support for M, KJ/KG and J/KG. Cannot find where check/conversion is. Comments?

Comment (TeeeJay) [reply src/libecalc/core/graph_result.py:700]: src/libecalc/input/mappers/utils.py:89

Comment (TeeeJay) [reply src/libecalc/core/graph_result.py:700]: e.g. from src/libecalc/input/mappers/model.py:127 and :187

**Review by jsolaas — APPROVED** [review]

**Pull request merged**